src/textual/widgets/_progress_bar.py

```diff
@@ -2,22 +2,43 @@
 
 from __future__ import annotations
 
-from math import ceil
+from math import ceil, cos, radians
 from time import monotonic
 from typing import Callable, Optional
+from typing_extensions import Literal
 
 from rich.style import Style
 
 from textual.geometry import clamp
 
 from ..app import ComposeResult, RenderResult
 from ..containers import Horizontal
+from ..css._error_tools import friendly_list
 from ..reactive import reactive
 from ..renderables.bar import Bar as BarRenderable
 from ..timer import Timer
 from ..widget import Widget
 from ..widgets import Label
 
+from ..color import Color
+
+
+BarColorScheme = Literal["default", "rainbow"]
+"""The names of the valid bar color_shemes.
+
+These are the color_shemes that can be used with a [`Bar`][textual.widgets.Bar].
+"""
+
+_VALID_BAR_COLORSCHEMES = {"default", "rainbow"}
```

Comment:
You've created variables like `BarColorScheme` that correctly capitalise `Scheme` because “color scheme” is not a single word in English.
Likewise, this should be:

```suggestion
_VALID_BAR_COLOR_SCHEMES = {"default", "rainbow"}
```

There are other instances of this issue in the code. I won't mark all of them explicitly.

Reply:
Fixed.